package/linux/dfhack

```diff
@@ -65,7 +65,7 @@ case "$1" in
     ret=$?
     ;;
   *)
-    setarch i386 -R env LD_PRELOAD=$PRELOAD_LIB ./libs/Dwarf_Fortress "$@"
+    setarch i386 -R env LD_PRELOAD="$PRELOAD_LIB" ./libs/Dwarf_Fortress "$@"
```

Comment:
You should add quotes around `$PRELOAD_LIB` to the lines above as well (otherwise this will prevent DFHack from being loaded when using a debugger).